server/src/services/album.service.ts

```diff
@@ -262,11 +261,25 @@ export class AlbumService {
       await this.access.requirePermission(auth, Permission.ALBUM_SHARE, id);
     }
 
-    await this.albumRepository.update({
-      id: album.id,
-      updatedAt: new Date(),
-      sharedUsers: album.sharedUsers.filter((user) => user.id !== userId),
-    });
+    await this.albumPermissionRepository.delete(userId, id);
+  }
+
+  async setAlbumPermission(
```

Comment:
Similar to the controller, this could in theory have its own service. Question is does it need to?

Reply:
Can you remember this to update permission?